src/bin/cli/app.rs

```diff
@@ -76,10 +76,10 @@ impl App {
         name: Option<String>,
         outdir: Option<PathBuf>,
     ) -> Result<()> {
-        let bin = util::path_to_str(&bin)?;
-        let chain = util::path_to_str(&chain)?;
+        let bin = util::path_to_string(&bin)?;
+        let chain = util::path_to_string(&chain)?;
```

Comment:
The renaming has something to do? 
Otherwise, I would prefer a separate commit for this, so it's clearer the purpose of each commit.